### PR TITLE
add cvx mappings table

### PR DESCRIPTION
**Pull request by ilaflamme-va**

### CODE REVIEW

**Review by KevinCranmer — APPROVED** [review]

**Review by bcastillomartel-va — DISMISSED** [review]

File: db/migration/V29_0__create_immunization_cvx_mappings.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* 
+ * Create Implantable Device Table.  
+ */
+
+CREATE TABLE [App].[CVX_VaccineGroup](
+  [CVX] [varchar](50) NULL,
+	[ShortDescription] [varchar](150) NULL,
+	[VaccineStatus] [varchar](10) NULL,
+	[VaccineGroupName] [varchar](25) NULL,
+    [CVXForVaccineGroup] [varchar](50) NULL
```
Comment (jblefkove-va) [inline db/migration/V29_0__create_immunization_cvx_mappings.sql:10]: tabs and inconsistent indentation

File: db/migration/V29_0__create_immunization_cvx_mappings.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    )
+GO
+
+CREATE index [IX_VaccineGroup_CVX] on [App].[CVX_VaccineGroup] ([CVX])
```
Comment (jblefkove-va) [inline db/migration/V29_0__create_immunization_cvx_mappings.sql:14]: For consistency, ``CREATE INDEX [IX_VaccineGroup_CVX] ON [App].[CVX_VaccineGroup]([CVX])`

File: minimart-manager/src/test/java/gov/va/api/health/minimartmanager/MinimartMaker.java [hunk 3]
```diff
@@ -620,11 +632,6 @@ public int deleteAllSpecialtiesByPractitionerRole(CompositeCdwId praRolCdwId) {
     return em.createQuery(query).executeUpdate();
   }
 
-  @SneakyThrows
-  private String fileToString(File file) {
```
Comment (jblefkove-va) [inline minimart-manager/src/test/java/gov/va/api/health/minimartmanager/MinimartMaker.java:624]: 🚮

File: datamart/db-dumps/ImmunizationCVXMappings.csv [hunk 4]
```diff
@@ -0,0 +1,209 @@
+CVX,ShortDescription,VaccineStatus,VaccineGroupName,CVXForVaccineGroup
+207,"COVID-19, mRNA, LNP-S, PF, 100 mcg/0.5mL dose or 50 mcg/0.25mL dose",Active,COVID-19,213
```
[the remark this answers is not in the document]
Comment (ilaflamme-va) [reply datamart/db-dumps/ImmunizationCVXMappings.csv:2]: This is how it was provided.

**Review by jblefkove-va — DISMISSED** [review]

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**